BusinessModels/GeneSIS.md

```diff
@@ -23,7 +23,7 @@ This page contains the business model for the GeneSIS (Orchestration and deploym
     </td>
     <td rowspan="1">
       <p>
-        Needs continuous deployment over IoT, Edge and Cloud. Need support for trustworthy deployment (i.e., support for deploying trustworthiness mechanisms and Mechanisms to properly monitor what has been deployed)
```

Comment:
The development and operation of smart IoT systems need support for **continuous** deployment over ALL different layers of IoT, Edge, and Cloud infrastructures. This includes deployment over local resources (including in local area network) with no direct access to the Internet. Moreover, the complexity of such smart IoT systems and the uncertainty of operating environments (e.g., evolving security threats) urge for **trustworthy** deployment (i.e., support for deploying trustworthiness mechanisms such as security/privacy controls, and mechanisms reliable deployment such as to properly monitor what has been deployed)